src/main/java/com/programmers/VoucherManagementApplication/voucher/FixedVoucher.java

```diff
@@ -0,0 +1,24 @@
+package com.programmers.VoucherManagementApplication.voucher;
+
+import com.programmers.VoucherManagementApplication.io.Message;
+import com.programmers.VoucherManagementApplication.vo.Amount;
+import com.programmers.VoucherManagementApplication.vo.VoucherType;
+
+import java.util.UUID;
+
+public class FixedVoucher extends Voucher {
+
+    private static final long MAX_VOUCHER_AMOUNT = 10000;
+
+    public FixedVoucher(UUID voucherId, VoucherType voucherType, Amount amount) {
+        super(voucherId, voucherType, amount);
+        if (amount.getAmount() > MAX_VOUCHER_AMOUNT) {
+            throw new IllegalArgumentException(Message.INVALID_FIXED_MAX.getMessage());
+        }
+    }
+//    @Override
```

Comment:
Fixed 와 Percent의 출력을 다르게 가져가기 위해서는
객체의 toString을 활용할 수 있을 것 같습니다.

```suggestion
	
	@Override
	public String toString() {
		return "FixedVoucher{" +
			"voucherId=" + voucherId +
			", voucherType=" + voucherType +
			", amount=" + amount +
			'}';
	}
```